src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -227,7 +227,16 @@ bool ReconstructionEngine_sequentialSfM::process()
 
     exportStatistics(elapsedTime);
 
-    return !_sfmData.getPoses().empty();
+    int nbviews = _sfmData.getViews().size();
+    int nbposes = _sfmData.getPoses().size();
+
+    int minPoses = 1;
+    if (nbviews > 5)
```

Comment:
I would a new param to the set of parameters of the class https://github.com/alicevision/AliceVision/blob/396d82cb1e4645a9cbd40b90eac920b2e73660b6/src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.hpp#L41
(with a line of comment that explains the meaning)
Not necessary that it is exposed, but it's better than having a random number in the code.